src/guide/render-function.md

```diff
@@ -567,6 +567,33 @@ render () {
 }
 ```
 
+## Return Values for Render Functions
+
+In all of the examples we've seen so far, the `render` function has returned a single root VNode. However, there are alternatives.
+
+Returning a string will create a text VNode, without any wrapping element:
+
+```js
+render() {
+  return 'Hello world!'
+}
+```
+
+We can also return an array of children, without wrapping them in a root node. This creates a fragment:
+
+```js
+// Equivalent to a template of `Hello<br>world!`
+render() {
+  return [
+    'Hello',
+    h('br'),
+    'world!'
+  ]
+}
+```
+
+If a component needs to render nothing, perhaps because data is still loading, it can just return `null`. This will be rendered as a comment node in the DOM.
```

Comment:
**suggestion**: should we mention that we can return `null` as well for renderless components?

Reply:
Renderless components aren't currently explained anywhere else in the docs. I don't think we can mention them here without several more sentences explaining what they are.

I'd rather leave this how it is for now.